7a_temp_coop_munge/src/data_parsers/parse_missouri_files.R

```diff
@@ -0,0 +1,85 @@
+
+# Parse the ~300 files within the `Bull_Shoals_Lake_DO_and_Temp.zip` file
+
+parse_Bull_Shoals_Lake_DO_and_Temp <- function(inind, outind) {
+
+  infile <- sc_retrieve(inind, remake_file = '6_temp_coop_fetch_tasks.yml')
+  outfile <- as_data_file(outind)
+
+  # unzip the file with all of Bull Shoals
+  files_from_zip <- unzip(infile, exdir = tempdir())
```

Comment:
Not a big deal since the tempdir gets wiped when you close the R session, but might want to get in the habit of doing clean up of the files you extract after you make use of them.

Could do this by using `on.exit()` with a delete/unlink of the unzipped directory whenever the function exists (either due to completing or hitting an error) or using unlink at the end of the function.